parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -94,17 +94,22 @@ installStandaloneContainerd() {
     CONTAINERD_VERSION=$1
     # azure-built runtimes have a "+azure" suffix in their version strings (i.e 1.4.1+azure). remove that here.
     CURRENT_VERSION=$(containerd -version | cut -d " " -f 3 | sed 's|v||' | cut -d "+" -f 1)
+    CURRENT_COMMIT=$(containerd -version | cut -d " " -f 4)
     # v1.4.1 is our lowest supported version of containerd
     
+    # we always default to the .1 patch version
+    CONTAINERD_PATCH_VERSION="1"
+
     #if there is no containerd_version input from RP, use hardcoded version
     if [[ -z ${CONTAINERD_VERSION} ]]; then
-        CONTAINERD_VERSION="1.4.8"
-        echo "Containerd Version not specified, using default version: ${CONTAINERD_VERSION}"
+        CONTAINERD_VERSION="1.4.9"
+        CONTAINERD_PATCH_VERSION="3"
+        echo "Containerd Version not specified, using default version: ${CONTAINERD_VERSION}-${CONTAINERD_PATCH_VERSION}"
     else
-        echo "Using specified Containerd Version: ${CONTAINERD_VERSION}"
+        echo "Using specified Containerd Version: ${CONTAINERD_VERSION}-${CONTAINERD_PATCH_VERSION}"
     fi
 
-    if semverCompare ${CURRENT_VERSION:-"0.0.0"} ${CONTAINERD_VERSION}; then
+    if [ semverCompare ${CURRENT_VERSION:-"0.0.0"} ${CONTAINERD_VERSION} ] && [ "${CURRENT_VERSION}" != "${CONTAINERD_VERSION}" ]; then
```

Comment:
please, _someone_ sanity check my bash.

Reply:
I ended up removing this because we can't check the inbound commit/revision of containerd without unpacking it fully anyway. we'll rely on VHD versions for same-patch revisions.